docs/data/data-grid/row-updates/row-updates.md

```diff
@@ -40,6 +40,11 @@ In addition, the area in which `onRowsScrollEnd` is called can be changed using
 
 {{"demo": "InfiniteLoadingGrid.js", "bg": "inline", "disableAd": true}}
 
+:::info
+Note that for infinite loading to work properly, sorting and filtering should be done on the server side.
+Otherwise, the sorting and filtering will be applied to loaded rows only.
```

Comment:
```suggestion
For sorting and filtering to work properly with the infinite loading, they should be applied on the server side.
Otherwise, the sorting and filtering will be applied to the partially loaded rows only.
```